pootle/settings/90-local.conf.template

```diff
@@ -60,6 +60,11 @@ DATABASES = {
         # See https://docs.djangoproject.com/en/1.6/topics/db/transactions/
         # required for Django 1.6 + sqlite
         'ATOMIC_REQUESTS': True,
+        # Note: if your database terminates idle connections after some time,
+        # you should set CONN_MAX_AGE to a lower value. Persistent connections
+        # where CONN_MAX_AGE = None can not be used.
+        # See https://docs.djangoproject.com/en/1.7/ref/databases/#connection-management
+        'CONN_MAX_AGE': 0,
```

Comment:
Shouldn't this also be included in `20-backends.conf` and `90-dev-local.conf.sample`?